drivers/SmartThings/sonos/src/api/sonos_connection.lua

```diff
@@ -225,7 +225,7 @@ function SonosConnection.new(driver, device)
         if body.version ~= favorites_version then
           favorites_version = body.version
 
-          local household = self.driver.sonos:get_household(header.householdId)
+          local household = self.driver.sonos:get_household(header.householdId) or { groups = {} }
```

Comment:
Would it be helpful to log an error when hitting the default case?

Reply:
I toyed with this idea, but didn't because the other branches dont. It is also only helpful if it goes to hub logs. I am hoping to get some more logs internally that can show me what is going on to make the household not available at this point. So not i dont think its particularly helpful.